src/sinks/util/http.rs

```diff
@@ -737,15 +793,17 @@ where
         Self {
             batch_service,
             _phantom: PhantomData,
+            _more_phantom: PhantomData,
         }
     }
 }
 
-impl<B, T: Send + 'static> Service<HttpRequest<T>> for HttpService<B, T>
+impl<B, T: Send + 'static, R: DriverResponse + Send + 'static + Sync> Service<HttpRequest<T>>
+    for HttpService<B, T, R>
```

Comment:
R must meet at least one of the following criteria:

- it appears in the implementing type of the impl, e.g. impl<T> Foo<T>
- for a trait impl, it appears in the implemented trait, e.g. impl<T> SomeTrait<T> for Foo
- it is bound as an associated type, e.g. impl<T, U> SomeTrait for T where T: AnotherTrait<AssocType=U>

R can not appear in the implemented trait as `Service` is defined in the `tower` crate

I have defined R with `HttpService`. However, `HttpService` must be Clone-able. **http::Response<T> is NOT cloneable...**

Reply:
Changing type of `http_response` to `&Response<Bytes> gets pretty messy